Description: Fix segfault in lookup_ldap initialisation
 lib/defaults.c doesn't initialise the config cache before lookup_ldap
 attempts to read from it.
Author: Malcolm Scott <debianpkg@malc.org.uk>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1503034

Index: autofs-5.1.1/modules/lookup_ldap.c
===================================================================
--- autofs-5.1.1.orig/modules/lookup_ldap.c
+++ autofs-5.1.1/modules/lookup_ldap.c
@@ -1665,6 +1665,11 @@ int lookup_init(const char *mapfmt, int
 		ctxt->check_defaults = 1;
 	}
 
+	if (!defaults_read_config(0)) {
+		free_context(ctxt);
+		return 1;
+	}
+
 	ctxt->timeout = defaults_get_ldap_timeout();
 	ctxt->network_timeout = defaults_get_ldap_network_timeout();
 
